mindmeld/components/nlp.py

```diff
@@ -770,9 +777,17 @@ def __init__(self, app_path, domain, resource_loader=None):
         super().__init__(app_path, resource_loader)
         self.name = domain
         self.intent_classifier = IntentClassifier(self.resource_loader, domain)
-        for intent in path.get_intents(app_path, domain):
+        intents = path.get_intents(app_path, domain)
+
+        # If there is only one intent in the domain, the classifier would not run
+        # hence we only account for classifiers were there are two or more intents
+        if len(intents) > 1:
+            self.progress_bar = progress_bar
+            self.progress_bar.total += 1
```

Comment:
We should increase total by the number of intents right?

Reply:
We dont technically train a classifier if there is only 1 intent.